[11.0][ADD] sale_order_invoicing_queued: Enqueue sales order invoicing (#588)
OCA/account-invoicing: sale_order_invoicing_queued/readme/DESCRIPTION.rst

This module allows to enqueue in several jobs the sales orders invoicing

> process to be executed in paralell on background, that it's normally done

Reviewer: ```suggestion
process to be executed in paralell on background, which is normally done
```

> process to be executed in paralell on background, that's it's normally done

Reviewer: ```suggestion
process to be executed in paralell on background, that is usually done
```

Author: OK about `'s`, but I have chosen normally because the etymological difference about what is "the rule": https://www.google.com/search?client=ubuntu&channel=fs&q=normally+vs+usually&ie=utf-8&oe=utf-8
